### DISCUSSION TITLE
how to use “/etc/hosts” to resolve host when using http3_client.py

### DISCUSSION THREAD

**Discussion by kuninchengdu** [aiortc/aioquic#304] [Q&A]
how to use “/etc/hosts” to resolve host when using http3_client.py

**Comment by devkdas** [comment]
i haven't tried with http3_client.py but i have tried with interop_test.py and able to get the output correctly
Place this below server in "interop.py" under "SERVERS=[ " section
Server("somename", "name_given_in_hosts_file", port=4433, verify_mode=ssl.CERT_NONE),
and run you own interop testcase using
python3 aioquic/examples/interop_test.py --server somename --test test_case_name -v

Hope this helps

**Comment by jlaine** [comment]
I'm not sure I understand the issue, the system's resolver should be picking up your entry in /etc/hosts?

What exactly did you add to /etc/hosts ?
